doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -80,11 +80,14 @@ described elsewhere
 
 .. cmd:: Check @term
 
-   Displays the type of :n:`@term`. When called in proof mode, the
-   term is checked in the local context of the selected goal.
+   Displays the type of :n:`@term`. When called in proof mode, the term is
+   checked in the local context of the selected goal. This command will try
+   resolve an existential variables.
 
 .. cmd:: Type @term
-   :undocumented:
+
+   Like :cmd:`Check`, displays the type of :n:`@term` however without attempting
+   to resolve existential variables.
```

Comment:
Why do you think that it doesn't resolve existentials?
~~~coq
Class A := {}.
Local Declare Instance Ainst : A.

Type _ : A. (* Ainst : A *)
~~~

Type is like Check, but always uses the global env (Check uses the first focused goal env when there is one) and errors if there are unresolvable existentials.

Reply:
What about typeclass search? I seem to recall `Type` not going into typeclass search. Maybe I need to dig in the sources to see what is going on.

Reply:
The example I posted is typeclass search

Reply:
> Why do you think that it doesn't resolve existentials?

Probably because you wrote "you can use Type instead of Check to forbid evars" when you posted the [comment](https://coq.zulipchat.com/#narrow/stream/237977-Coq-users/topic/Declaring.20record.20constructor.20as.20Canonical/near/277299748) that prompted Ali to create #15881.

Reply:
@SkySkimmer is correct however. I've given an example displaying the difference in the reply to jfehrle above.